script/packagecloud.rb

```diff
@@ -43,6 +43,7 @@
     "sles/15.2",  # Current
   ],
   "centos/8" => [
+    "almalinux/8",
```

Comment:
This looks fine.  This just means that when we build the CentOS 8 package, we'll upload the exact same artifact for AlmaLinux 8, which is probably what we wanted here.